src/applications/personalization/profile-2/components/ProfileInfoTable.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ProfileInfoTable = ({ data, fieldName, title }) => (
+  <table className="profile-info-table" data-field-name={fieldName}>
+    <thead>
+      <tr>
+        <th>
+          <h3>{title}</h3>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      {data.map((row, index) => (
+        <tr key={index}>
+          <td>
+            <h4 className="vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--bold">
+              {row.title}
+            </h4>
+          </td>
+          <td>{row.value}</td>
+        </tr>
+      ))}
+    </tbody>
+  </table>
```

Comment:
Totally optional, but instead of requiring the data to be passed in a formatted state, it might be cool to have this component accept a converter function as a prop similar to a custom `render` prop. It would probably only be warranted if there was custom JSX returning from it though.

```jsx
  {data
    .map(element => convertToRow ? convertToRow(element) : element)
    .map((row, index) => (
        <tr key={index}>
          <td>
            <h4 className="vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--bold">
              {row.title}
            </h4>
          </td>
          <td>{row.value}</td>
        </tr>
      ))}
```

Could also be useful for debugging to have the original data structure in state.

Reply:
I think I like that idea.